tests/Uri/test_uri.py

```diff
@@ -0,0 +1,56 @@
+import pytest
+from smsdk import client
+from smsdk.utils import get_url
+
+
+def test_uri() -> None:
+    tenant: str = ""
+    cli: client.Client = client.Client(tenant)
+
+    assert cli.tenant == tenant
+    assert cli.config["protocol"] == "https"
+    assert cli.config["site.domain"] == "sightmachine.io"
+
+    tenant = "demo"
+    cli = client.Client(tenant)
+
+    assert cli.tenant == "demo"
+    assert cli.config["protocol"] == "https"
+    assert cli.config["site.domain"] == "sightmachine.io"
+
+    tenant = "demo-sdk-test.localnet"
+    cli = client.Client(tenant)
+
+    assert cli.tenant == "demo-sdk-test"
+    assert cli.config["protocol"] == "https"
+    assert cli.config["site.domain"] == "localnet"
+
+    tenant = "://demo-sdk-test.localnet"
+    cli = client.Client(tenant)
+
+    assert cli.tenant == "demo-sdk-test"
+    assert cli.config["protocol"] == "https"
+    assert cli.config["site.domain"] == "localnet"
+
+    tenant = "http://demo-sdk-test"
+    cli = client.Client(tenant)
+
+    assert cli.tenant == "demo-sdk-test"
+    assert cli.config["protocol"] == "http"
+    assert cli.config["site.domain"] == "sightmachine.io"
+
+    tenant = "https://demo-sdk-test.localnet"
+    cli = client.Client(tenant)
+
+    assert cli.tenant == "demo-sdk-test"
+    assert cli.config["protocol"] == "https"
+    assert cli.config["site.domain"] == "localnet"
+
+    cli = client.Client("http://demo-sdk-test.localnet:8080/")
+
+    tenant = "http://demo-sdk-test.localnet:8080/"
```

Comment:
How do we verify the port used in requests?

Reply:
In the current implementation, there is no support for port verification. Do we need to verify it?

Reply:
In general, I doubt a user should need to set the port for a http/https URI, so may want to warn that is wil be ignored.  But for an experiment, I can suggest port-forwarding the webapp service and point the SDK at that.  That can be any port then.

Reply:
Ignoring the user provided port in the URL